### PR TITLE
[IMP] accounting: employment hero added taxes mapping descriptions

### PR DESCRIPTION
**Pull request by tong-odoo**
Impacted Versions:

- saas-17.1 and above

### CODE REVIEW

**Review by dade-odoo — APPROVED** [review]
Hey @tong-odoo thanks for forward porting to keep all the versions up to date! We're trying to reduce the number of screenshots where it's possible for a few reasons (they increase the load on the documentation site and they necessitate more frequent updates as there are minor UI changes).

If it is a necessary screenshot, that's okay! In that case, we're trying to minimize the red markups on screenshots as they are unnecessary, and emphasis on fields can be accomplished through effective cropping, and the explanation should be clear enough even without the screenshot.

I'll go ahead and approve and request the final review :) thanks!

**Comment by tong-odoo** [comment]
@dade-odoo Thanks for the review! I will remove the screenshot then.

File: content/applications/finance/fiscal_localizations/employment_hero.rst [hunk 1]
```diff
@@ -46,6 +46,9 @@ Configuration
         :alt: The Employment Hero "Business ID" number is in the URL
 
    - You can choose any Odoo journal to post the payslip entries.
+#. Configure the tax by going to :menuselection:`Accounting --> Configuration --> Taxes`. Create the
+   necessary taxes for the Employment Hero payslip entries. Fill in the tax code from
+   **Employment Hero** to :guilabel:`Matching Employment Hero Tax`.
```
Comment (auva-odoo) [inline content/applications/finance/fiscal_localizations/employment_hero.rst:51]:
```suggestion
   **Employment Hero** in the :guilabel:`Matching Employment Hero Tax` field.
```

**Review by auva-odoo — CHANGES_REQUESTED** [review]
Hi @tong-odoo, just one small change. Thank you for your work!

**Review by auva-odoo — APPROVED** [review]
Thanks @tong-odoo! You can r+ whenever you're ready :slightly_smiling_face: 
@robodoo delegate+

**Comment by tong-odoo** [comment]
@robodoo r+

**Comment by tong-odoo** [comment]
Thank you @auva-odoo for your review. Could you take a look at this PR as well? This is basically the same but older version is using Australia to show the employment hero information.
https://github.com/odoo/documentation/pull/9339

**Comment by vin-odoo** [comment]
Note: had to rebase

**Comment by tong-odoo** [comment]
@robodoo r+

**Pull request closed without merging**